spec/truffle/keyword_arguments.rb

```diff
@@ -0,0 +1,201 @@
+# truffleruby_primitives: true
```

Comment:
Because this file doesn't have a `_spec.rb` suffix it won't be run e.g. for `jt test fast` or `jt test :truffle`, so should be renamed.

Reply:
@tomstuart